pubsublite/types.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+package pubsublite
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// LocationPath stores a path consisting of a project and zone.
+type LocationPath struct {
+	// A Google Cloud project. The project ID (e.g. "my-project") or the project
+	// number (e.g. 987654321) can be provided.
```

Comment:
nit: perhaps use the quoted string for project number for explicitiness
```suggestion
	// number (e.g. "987654321") can be provided.
```

Reply:
Done